src/_repobee/fileutil.py

```diff
@@ -38,3 +43,60 @@ def __init__(self, label: str):
 
     def __str__(self):
         return str(self.label)
+
+
+def find_files_by_extension(
+    root: Union[str, pathlib.Path], *extensions: str
```

Comment:
You said that you prefer unqualified names in type hints. Should we also change `pathlib.Path` to `Path`?

Reply:
It's not that important, but yes in general I think type hints should have unqualified names to make them as clutter-less as possible.